templates/docs/examples/patterns/image/container/aspect-ratio/16-9.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image Container / Aspect Ratio / 16/9{% endblock %}
+
+{% block standalone_css %}patterns_image{% endblock %}
+
+{% block content %}
+<div class="p-image-container--16-9 u-align--center u-vertically-center">
+  <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,c_fill,w_720/https://assets.ubuntu.com/v1/0ecd2ffd-community_wallpaper_noble_little_numbat_boy_by_azskalt.jpeg" alt="">
```

Comment:
I think it's OK to use direct `https://assets.ubuntu.com/v1/...` links in Vanilla examples. Cloudinary links are usually generated by our image template, so I'd rather not hardcode them into examples.
And it will be also good to avoid using third party service to serve these images as this can impact on Percy snapshots being less stable.

Reply:
Revised to use assets API directly